src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function cyclingFunc(param) {
```

Comment:
you can use '...args' instead of 'param' and and check if parameters are passed using if (args.length > 0).
I think this is not a mistake, just a more general way